seapath_ros_driver/include/seapath_ros_driver.hpp

```diff
@@ -0,0 +1,103 @@
+#ifndef SEAPATH_DRIVER_H
+#define SEAPATH_DRIVER_H
+
+#include <iostream>
+#include <sstream>
+#include <cmath> 
+#include <vector>
+#include <chrono>
+#include <memory>
+
+#include <rclcpp/rclcpp.hpp> 
+#include <sensor_msgs/msg/nav_sat_fix.hpp>
+#include <geometry_msgs/msg/twist_with_covariance_stamped.hpp>
+#include <geometry_msgs/msg/pose_with_covariance_stamped.hpp>
+#include <tf2/transform_datatypes.h>
+#include <tf2/LinearMath/Quaternion.h>
+#include <diagnostic_msgs/msg/diagnostic_status.hpp>
+#include <vortex_msgs/msg/KMBinary.hpp>
+
+#include <seapath_socket.hpp>
```

Comment:
Seapath socket is a local header file in your package, not an external library, and should actually therefore use "".